src/amber/support/message_encryptor.cr

```diff
@@ -24,22 +28,30 @@ module Amber::Support
     # Verify and Decrypt a message. We need to verify the message in order to
     # avoid padding attacks. Reference: http://www.limited-entropy.com/padding-oracle-attacks.
     def verify_and_decrypt(value : String) : Bytes
-      decrypt(verifier.verify_raw(value))
+      verify_and_decrypt(Base64.decode(value))
```

Comment:
The base64 de-marshaling here doesn't match the marshaling that is happening [when the cookie is written](https://github.com/amberframework/amber/pull/404/files#diff-e135a6eb6d4cfcdee2e8019a8f4a005aR257), does it make sense to have this class handle both Base64 encoding and decoding?

Reply:
I'm only trying to point out that by my reading, the code flow for decrypting a message is:

other->MessageEncryptor->base64.decode

and the code flow for encrypting is:

other->base64.encode->MessageEncryptor

I'd rather MessageEncryptor handle both base64 directions or neither.

Reply:
Neither makes more sense, since method overloading doesn't work with return values, we'd have to create another method called encrypt_and_base64_encode or something.

I think removing it makes more sense.